### ISSUE TITLE
Not clear what units are used.

### ISSUE THREAD

**Issue by mivoligo** [Feichtmeier/settings#306]
In Privacy>House Keeping
![Screenshot from 2022-03-16 20-16-21](https://user-images.githubusercontent.com/1629120/158682855-d3b37477-0a19-4e5c-9fa0-cedf9e8ef491.png)

I assume it's in days but maybe we can make it more obvious.

Similar situation here:
In Privacy > Screen Lock
![Screenshot from 2022-03-16 20-18-05](https://user-images.githubusercontent.com/1629120/158683092-f6eb51a4-2f66-47e7-922e-1b3e72d80238.png)

There's also number of missing units in A11y page. They are mostly about delay times for clicking and such.

**Comment by mivoligo** [comment]
Partial fix in #307 Not sure how we should handle "Period for automatic deletion". Should we add "in days" in subtitle, should we use dropdown for selecting the time, or should we just skip it for now as a user is smart enough to know it's in days?

**Comment by Feichtmeier** [comment]
> Partial fix in #307 Not sure how we should handle "Period for automatic deletion". Should we add "in days" in subtitle, should we use dropdown for selecting the time, or should we just skip it for now as a user is smart enough to know it's in days?

Good question. Do what you think is most intuitive please :)
